packages/block-library/src/file/edit.js

```diff
@@ -171,13 +168,19 @@ function FileEdit( {
 				} }
 			/>
 			<BlockControls>
-				<MediaReplaceFlow
-					mediaId={ id }
-					mediaURL={ href }
-					accept="*"
-					onSelect={ onSelectFile }
-					onError={ onUploadError }
-				/>
+				<ToolbarGroup>
+					<MediaReplaceFlow
+						mediaId={ id }
+						mediaURL={ href }
+						accept="*"
+						onSelect={ onSelectFile }
+						onError={ onUploadError }
+					/>
+					<ClipboardToolbarButton
+						text={ href }
+						disabled={ isBlobURL( href ) }
```

Comment:
I don't see this prop being used in the component

Reply:
i added it as prop in daddou-ma/gutenberg@ebb7fee